.travis.yml

```diff
@@ -7,10 +7,9 @@ script:
 sudo: false
 
 rvm:
-  - 2.0.0
   - 2.1.9
-  - 2.3.1
-  - 2.4.1
+  - 2.4.4
```

Comment:
Why switch away from ruby versions that Puppet uses? I get dropping the old versions, though this should use 2.4.3 to match the latest Puppet 5 release.

https://puppet.com/docs/puppet/5.5/about_agent.html

Reply:
I can add 2.4.3 in, I added 2.4.4 based on what PDK uses. The official PDK installation uses Ruby 2.4.4 with Puppet 5.5.1